vg/vg.go

```diff
@@ -85,6 +88,49 @@ type CanvasSizer interface {
 	Size() (x, y Length)
 }
 
+// CanvasWriterTo wraps behavior required for writing
+// a Canvas representation to a concrete image.
+type CanvasWriterTo interface {
+	CanvasSizer
+	io.WriterTo
+}
+
+var (
+	formatLock sync.RWMutex
+	formats    = make(map[string]func(w, h Length) CanvasWriterTo)
```

Comment:
depends on the use case, I suppose.
for articles plots, yeah, they are the "odd cousins".
for interactive plots, they're definitely the more useful and common formats.

speaking of which: it seems the naming has some cognitive dissonance. you speak of formats, I speak (as per my "vg/drivers" thread on gonum-dev) of drivers.
formats does hint at a disk or persistent representation.
drivers is more generic (which may or may not be a good thing)